src/test/javascript/spec/component/course/image-cropper-modal.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ImageCropperModalComponent } from 'app/course/manage/image-cropper-modal.component';
+
+describe('ImageCropperModalComponent', () => {
+    let component: ImageCropperModalComponent;
+    let fixture: ComponentFixture<ImageCropperModalComponent>;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            imports: [ImageCropperModalComponent],
+        }).compileComponents();
+
+        fixture = TestBed.createComponent(ImageCropperModalComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
+    });
+});
```

Comment:
Write a proper test here with the items we discussed

Reply:
Tests are added